libs/virtual-node/virtual-node-datamodel/src/main/kotlin/net/corda/libs/virtualnode/datamodel/entities/HoldingIdentityEntity.kt

```diff
@@ -64,4 +66,21 @@ data class HoldingIdentityEntity(
         this.uniquenessDDLConnectionId = uniquenessDDLConnectionId
         this.uniquenessDMLConnectionId = uniquenessDMLConnectionId
     }
+
+    fun toDTO(): HoldingIdentity {
```

Comment:
```suggestion
    fun toHoldingIdentity(): HoldingIdentity {
```
I think I saw the same for vnode info something so maybe we could be consistent